pkg/status/status.go

```diff
@@ -98,7 +98,6 @@ func GetStatus(verbose bool) (map[string]interface{}, error) {
 		stats["filterErrors"] = containers.GetFilterErrors()
 	}
 
-	stats["remoteConfiguration"] = getRemoteConfigStatus()
```

Comment:
I've moved this into the expvarcollector since the information is being gather using expvars